src/cards/wakatime-card.js

```diff
@@ -302,7 +303,10 @@ const renderWakatimeCard = (stats = {}, options = { hide: [] }) => {
             return createTextNode({
               id: language.name,
               label: language.name,
-              value: language.text,
+              value:
+                display_format == "percent"
+                  ? language.percent.toString()
```

Comment:
@ritik48, thanks for your pull request. For the percentage case, let's add the `%` suffix like is done in the language card. I will add the `hacktoberfest-accepted` label when that change is done 👍🏻.

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)